service/command.go

```diff
@@ -16,12 +16,58 @@ package service // import "go.opentelemetry.io/collector/service"
 
 import (
 	"errors"
+	"fmt"
 
 	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+
+	"go.opentelemetry.io/collector/config"
 
```

Comment:
Those two imports should be together (`go.opentelemetry.io/collector/...`) but I think the linter will tell you that as well :-)

Reply:
It does

Reply:
Thanks I attempted putting them together but when I ran "make fmt", it sorted them back in this order